package.json

```diff
@@ -52,6 +52,8 @@
     "@stoplight/spectral-functions": "^1.5.1",
     "@stoplight/spectral-parsers": "^1.0.1",
     "@stoplight/spectral-rulesets": "^1.4.3",
+    "@types/json-schema": "^7.0.11",
```

Comment:
Packages from `@types/` should be installed in the dev-dependencies.

Reply:
Good catch. Fixed!